synapse/lib/schemas.py

```diff
@@ -668,9 +668,22 @@
             'pattern': s_version.semverstr,
         },
         'build': {
-            'type' 'object'
+            'type': 'object',
             'properties': {
                 'time': {'type': 'number'},
+                'synapse:verstring': {
+                    'type': 'string',
+                    'pattern': s_version.semverstr
+                },
+                'synapse:version': {
+                    'type': 'array',
```

Comment:
you could specify that this requires a length of 3 just to be super strict

Reply:
I didn't want to constrain it too much in case we decide to add additional version specifiers to the end.